src/altair/validator.rs

```diff
@@ -1,8 +1,7 @@
+use crate::altair::SYNC_COMMITTEE_SUBNET_COUNT;
 use crate::primitives::{BlsSignature, Root, Slot, ValidatorIndex};
 use ssz_rs::prelude::*;
 
-pub const SYNC_COMMITTEE_SUBNET_COUNT: usize = 4;
```

Comment:
id keep this here to mirror the `consensus-specs` and then import in the other place